src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/EncapsulateField_OutOfProc.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Windows.Automation;
+using Microsoft.VisualStudio.IntegrationTest.Utilities.InProcess;
+using Microsoft.VisualStudio.IntegrationTest.Utilities.Input;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+
+namespace Microsoft.VisualStudio.IntegrationTest.Utilities.OutOfProcess
+{
+    public class EncapsulateField_OutOfProc : OutOfProcComponent
+    {
+        private const string dialogName = "Preview Changes - Encapsulate Field";
+
+        public EncapsulateField_OutOfProc(VisualStudioInstance visualStudioInstance) : base(visualStudioInstance)
+        {
+        }
+
+        public void Invoke()
+                => VisualStudioInstance.Editor.SendKeys(new KeyPress(VirtualKey.R, ShiftState.Ctrl), new KeyPress(VirtualKey.E, ShiftState.Ctrl));
+
+        public void VerifyOpen()
```

Comment:
I'd imagine not even having `EncapsulateField_OutOfProc`, just having a separate way to do the Invoke(), and then use `PreviewChanges_OutOfProc` directly in your tests. I think it will make the tests more readable _and_ remove a bunch of forwarding methods.